### ISSUE TITLE
Debugger: `@enter` does not work with macro

### ISSUE THREAD

**Issue by hdavid16** [julia-vscode/julia-vscode#2710]
The following does not work:

```julia
using JuMP
m=Model()
@enter @variable(m,x)
```

For some reason, the debugger evaluates the expression as `@variable x` (it misses the model `m`, causing an `UndefVarError` exception to be thrown).

**Comment by pfitzseb** [comment]
Yeah, that's expected behaviour at the moment. Just wrap the macro call in a function.

**Comment by hdavid16** [comment]
Ok. Thanks!
